src/features/Bluetooth/Download/DownloadSlice.ts

```diff
@@ -163,16 +163,6 @@ function* tryDownloadForSensor({
       );
 
       yield call(downloadManager.saveLogs, sensorLogs);
-      if (sensorLogs.length) {
-        yield call(
-          btService.updateLogIntervalWithRetries,
```

Comment:
Was there another reason we did this, or just to clear the logs?
It looks like changing the interval always wipes the logs on the BLUE MAESTRO so the `clearLogs: false` parameter here doesn't actually work.

Reply:
The docs say "When changed, the current recorded data will be erased and logging will commence afresh." which doesn't explain the case when the interval is `60` and changed to `60` say. does that constitute a 'change'? I think it does.

why the interval is being set..? not entirely sure. I have a vague memory that the interval was defaulting back to a value. I would prefer to call `*clr` deliberately if the intention is to clear the logs.

the `clearLogs` prop is just for the Laird sensors - which do not automatically clear the logs when setting the interval. It would be good to split out that function, now that I look at it, and deliberately clear

Reply:
With Bluemaestro if you set the interval, it automatically wipes the logs.
I guess we should check if it's not correct and only update if it's not set already?